web/packages/teleport/src/services/integrations/integrations.ts

```diff
@@ -545,6 +546,28 @@ export const integrationService = {
         };
       });
   },
+
+  awsRolesAnywherePing(
+    integrationName: string,
+    trustAnchorArn: string,
+    syncRoleArn: string,
+    syncProfileArn: string
```

Comment:
i think it's safer to convert this to an object param instead since they are all strings and easy to mix up